### ISSUE TITLE
Repository - Clarification needed

### ISSUE THREAD

**Issue by schneiderpy** [marberts/piar#10]
Yesterday I was reviewing the suggestions mentioned [here](https://github.com/openjournals/joss-reviews/issues/6781#issuecomment-2158289330), in the main and default branch of the repository as mentioned in the initial review threat. However, I could not find any of them, including the status badge. I'm aware, that there's a different branch in the repo called JOSS. However, as far as I know, people use a different branch for the review with the paper.md file, and after the review most of them delete this branch.
JOSS review guide says:

>The paper may be in a short-lived branch which is never merged with the default, although if you do this, make sure this branch is created from the default so that it also includes the source code of your submission.  

Since this procedure is, in general, based on a temporal branch, my concern is that the main branch will not reflect the JOSS review process.
To avoid further confusion and perhaps unjustified "acceptance-blockers" I would like to receive some clarification on this @crvernon

As part of the [review ](https://github.com/openjournals/joss-reviews/issues/6781)

**Comment by crvernon** [comment]
@schneiderpy I currently have the default branch set as `joss` for the review.  When we finalize the review, we will get the authors to establish an archive (e.g., Zenodo record with a DOI) that is linked to a specific GitHub release.  This will ensure the permanence of any changes that may be made during this review.  This archive will also be tied directly to the paper upon publication.

**Comment by schneiderpy** [comment]
I didn't get the technical part ... what will be the default branch after the review? (the page that any user interested in the package will see?)

**Comment by marberts** [comment]
@schneiderpy Here's how I'm planning to incorporate the changes from this review. The `joss` branch contains all the work done for the paper and all the changes that have come from your comments and suggestions so far. I will continue to make changes in this branch as the review progresses. I've kept it separate from the `main` branch for now as `main` holds the stable development version of the package, and feeds to R-Universe.

Once the review is done, I'll bump the version from the current 0.8.0 to either 0.8.1 (if there are no breaking changes) or 0.9.0 (if there are significant changes) and merge into the `main` branch. All the commits in the `joss` branch will then be reflected in the history for `main`. I'll then make a release for this new version (which will automatically make the Zenodo record) and push the new version to CRAN and conda forge.

Hope that answers your question.

**Comment by crvernon** [comment]
Thanks @marberts - @schneiderpy I believe this should satisfy your question?  Let me know if not.  Thanks!

**Comment by schneiderpy** [comment]
Thank you @crvernon and @marberts. That makes things clearer.
